app/controllers/concerns/hyrax/works_controller_behavior.rb

```diff
@@ -199,6 +200,10 @@ def curation_concern_from_search_results
 
       # Only returns unsuppressed documents the user has read access to
       def search_result_document(search_params)
+        # Don't filter any models when fetching a single result.
+        search_builder.class.default_processor_chain.delete :filter_models
+        # Remove pagination params
+        search_params.delete :page
```

Comment:
What do you think about ` _, document_list = search_results(search_params.except(:page))` instead of removing the `:page` param altogether?

I'm :+1: on keeping this side-effect, since it was in the previous version, but also think it might be nice to avoid introducing it to this method. Existing callers may not expect the arguments to be mutated.